### ISSUE TITLE
SyntaxError on Python 3.5, with sshpubkeys 3.2.0+ 

### ISSUE THREAD

**Issue by bbtfr** [ojarva/python-sshpubkeys#81]
```
 /usr/local/lib/python3.5/dist-packages/sshpubkeys/__init__.py:2: in <module>
     from .keys import *  # pylint:disable=wildcard-import
 E     File "/usr/local/lib/python3.5/dist-packages/sshpubkeys/keys.py", line 406
 E       raise InvalidKeyError(f"Application string: {err}") from err
 E                                                        ^
 E   SyntaxError: invalid syntax
```

**Comment by ojarva** [comment]
Hi @bbtfr ,

When I saw this, I was wondering two things,

a) Why did I add f-strings and
b) Why didn't tests catch this.

Now that I got into checking, tests aren't executed with python3.5 because of https://www.python.org/downloads/release/python-3510/ - 3.5 is pretty much EOL'd.

Considering probably the only blocker for using 3.5 is f-strings which are easily replaced with `.format`, I'm happy to accept that as a PR but I personally consider 3.5 to be something that is no longer officially supported.
